kbase-extension/static/kbase/js/widgets/function_output/kbaseSeqCompView.js

```diff
@@ -86,14 +74,14 @@ define([
                         // kbws.get_object_subset([{ref: 'fangfang:1452395167784' +"/"+ 'Rhodobacter_CACIA_14H1_contigs', included: ['contigs/[*]/id', 'contigs/[*]/length', 'id', 'name', 'source', 'source_id', 'type']}], function(data) {
 
                         container.empty();
-                        var data = data[0].data;
-                        console.log(data);
+                        const dataData = data[0].data;
```

Comment:
`data` already used in upper scope, so I'm calling this `dataData` to reflect its origins